src/ros2_aruco/ros2_aruco/ros2_aruco/aruco_generate_marker.py

```diff
@@ -16,13 +16,27 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
     pass
 
 
+def addWhiteBorder(image, borderSize):
+    newSize = (image.shape[0] + borderSize * 2, image.shape[1] + borderSize * 2)
+
+    imageWithBorder = np.zeros(newSize, dtype=image.dtype)
+    imageWithBorder += 255
```

Comment:
Use np.full instead since we are changing all pixels to white anyway

Reply:
Do you mean setting all pixels to white then adding the aruco tag?

Reply:
Yea np.full(newSize, 255, dtype=image.dtype)

whats the point in creating a black one then iterating over it to turn it white? Just make a white one from the get go